packages/cashier/src/Containers/__tests__/account-transfer.spec.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { Router } from 'react-router';
+import { createBrowserHistory } from 'history';
+import AccountTransfer from '../account-transfer';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+jest.mock('@deriv/shared/src/services/ws-methods', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    WS: {
+        wait: (...payload) => {
+            return Promise.resolve([...payload]);
+        },
+    },
+}));
+
+jest.mock('Components/Form/account-transfer-form', () => jest.fn(() => 'mockedAccountTransferForm'));
+jest.mock('Components/Form/crypto-transactions-history', () => jest.fn(() => 'mockedCryptoTransactionsHistory'));
+jest.mock('Components/Error/cashier-locked', () => jest.fn(() => 'mockedCashierLocked'));
+jest.mock('Components/Receipt/account-transfer-receipt', () => jest.fn(() => 'mockedAccountTransferReceipt'));
+jest.mock('Components/Error/error', () => jest.fn(() => 'mockedError'));
+
+describe('<AccountTransfer />', () => {
+    const props = {
+        error: {},
+        onMount: jest.fn(),
+        recentTransactionOnMount: jest.fn(),
+        setAccountTransferAmount: jest.fn(),
+        setActiveTab: jest.fn(),
+        setIsTransferConfirm: jest.fn(),
+        setSideNotes: jest.fn(),
+    };
+
+    it('should render the account transfer form', async () => {
+        render(<AccountTransfer {...props} />);
+
+        await waitFor(() => {
+            expect(screen.getByText('mockedAccountTransferForm')).toBeInTheDocument();
+        });
+    });
+
```

Comment:
@farrah-deriv Isn't the bellow type of writing nicer?
`expect(await screen.findByText('mockedAccountTransferForm')).toBeInTheDocument();`
instead of 
`await waitFor(() => {
            expect(screen.getByText('mockedAccountTransferForm')).toBeInTheDocument();
        });`